### DISCUSSION TITLE
Separate configuration option for WebSocket address in livereload module

### DISCUSSION THREAD

**Discussion by wout** [luckyframework/lucky#1776] [Feature requests & Ideas]
It happens regularly that I have to monkey patch the `Lucky::LiveReloadTag` module to use another WebSocket address.

One scenario is with Ngrok, for example, which means using a Ngrok tunnel and `wss`. Another scenario is using docker, where the `server.host` option should be set to `"0.0.0.0"`, but the WebSocket should still be using `"127.0.0.1"`.

I'm not sure what's the best way to approach this. Probably with a fallback, if the live reload address is not configured?

**Comment by jwoertink** [comment]
Would it make sense to just add `reload_host` here https://github.com/luckyframework/lucky_cli/blob/main/src/web_app_skeleton/config/watch.yml#L3 ?

**Reply by wout** [reply]
But what about using `wss`? Or can the `reload_host` also be passed with the scheme?

**Reply by jwoertink** [reply]
`reload_host` doesn't exist... so it can do whatever we need it to do.
